scripts/deploy.sh

```diff
@@ -418,6 +448,15 @@ _config_uwsgi_centos() {
   sudo semodule -i nginx-uwsgi.pp
 }
 
+_config_uwsgi_opensuse() {
+  sudo cp -afv $scripts_path/uwsgi.service /etc/systemd/system/
+  sudo systemctl daemon-reload
+}
+
+_config_redis_opensuse() {
+  sudo cp -afv /etc/redis/default.conf.example /etc/redis/default.conf
```

Comment:
To allow parallel installation of different redis services, I would propose to use "/etc/redis/telemetrics.conf" here.